x-pack/plugins/ingest_manager/server/saved_objects.ts

```diff
@@ -75,4 +76,15 @@ export const savedObjectMappings = {
       },
     },
   },
+  [PACKAGES_SAVED_OBJECT_TYPE]: {
```

Comment:
I think we'll eventually want to define this inside an EPM section and then import that definition, but this is good for now.

Reply:
I'll address it in an issue I need to create for index patterns mappings which don't exist yet.